docs/source/visualisation/kedro-viz_visualisation.md

```diff
@@ -66,46 +66,80 @@ By convention, a [pipeline can be defined as having different layers](../resourc
 
 For example, the [data engineering convention](https://towardsdatascience.com/the-importance-of-layered-thinking-in-data-engineering-a09f685edc71) labels datasets according to the stage of the pipeline (e.g. whether the data has been cleaned).
 
-You can add a `layer` attribute to the datasets in the Data Catalog, which is reflected in the Kedro-Viz visualisation.
+As of Kedro version 18.9, we've made a change to the way layers are defined in the Data Catalog. Instead of specifying the layer attribute directly within a dataset's definition, it is now included under the kedro-viz metadata. 
 
```

Comment:
Please can you add a sentence to introduce the code to indicate this is the new specification. Something like

```suggestion
Here is how to include the layer definition using Kedro-Viz metadata:
```